README.md

```diff
@@ -67,86 +84,58 @@ The table below outlines what version of Nexus Repository the plugin was built a
-  $ ./nexus run
-  > bundle:install file:///tmp/nexus-repository-r-1.0.3.jar
-  ```
-  This will return text similar to "Bundle ID: 282".  You can double check this ID using bundle:list 
-  (for org.sonatype.nexus.plugins:nexus-repository-r; it should be the last one).  Then start it using this 
-  bundle command:
-  ```
-  > bundle:start <org.sonatype.nexus.plugins:nexus-repository-r ID>
-  ```
-
-### (more) Permanent Install
-
-For more permanent installs of the nexus-repository-r plugin, follow these instructions:
-
-* Copy the bundle (nexus-repository-r-1.0.3.jar) into <nexus_dir>/deploy
+### Permanent Reinstall
 
-This will cause the plugin to be loaded with each restart of Nexus Repository. As well, this folder is monitored
-by Nexus Repository and the plugin should load within 60 seconds of being copied there if Nexus Repository
-is running. You will still need to start the bundle using the karaf commands mentioned in the temporary install.
-
-### (most) Permanent Install
-
-If you are trying to use the R plugin permanently, it likely makes more sense to do the following:
-
-* Copy the bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-r/1.0.3/nexus-repository-r-1.0.3.jar`
+* Copy the new bundle into `<nexus_dir>/system/org/sonatype/nexus/plugins/nexus-repository-r/1.1.1-SNAPSHOT/nexus-repository-r-1.1.1-SNAPSHOT.jar`
 * If you are using OSS edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
 * If you are using PRO edition, make these mods in: `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-pro-feature/3.x.y/nexus-pro-feature-3.x.y-features.xml`
+
    ```
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
-   +     <feature version="1.0.3" prerequisite="false" dependency="false">nexus-repository-r</feature>
+   +     <feature version="1.1.1.SNAPSHOT" prerequisite="false" dependency="false">nexus-repository-r</feature>
          <feature version="3.x.y.xy" prerequisite="false" dependency="false">nexus-repository-yum</feature>
      </feature>
    ```
    And
    ```
-   + <feature name="nexus-repository-r" description="org.sonatype.nexus.plugins:nexus-repository-r" version="1.0.3">
+   + <feature name="nexus-repository-r" description="org.sonatype.nexus.plugins:nexus-repository-r" version="1.1.1.SNAPSHOT">
    +     <details>org.sonatype.nexus.plugins:nexus-repository-r</details>
-   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-r/1.0.3</bundle>
+   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-r/1.1.1-SNAPSHOT</bundle>
    + </feature>
     </features>
    ```
 This will cause the plugin to be loaded and started with each startup of Nexus Repository.
 
-## The Fine Print
+### Easiest Install for version  
+Relevant only for Nexus Repository manager version 3.18.0-01 and lower  which doest includes `R` plugin.
```

Comment:
does it relevant to 3.19 ?

Reply:
yes. but 3.19 is not released yet.

Reply:
I'd just put 3.19 now save yourself the work later.
Unless you plan to push this before 3.20 (when these changes are live).

Reply:
Typo: "doesn't include" (instead of "doest includes")

Reply:
50b34294944717d1e1b949b6813f0ecf45b1985f